### ISSUE TITLE
Promise rejection - path must be a string of Buffer

### ISSUE THREAD

**Issue by AmitMY** [automutate/autolesshint#62]
Running `autolesshint` or `autolesshint src/` results in the following:
```
(node:5864) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: path must be a string or Buffer
(node:5864) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Only thing I managed to run with no error is `-V`.

lesshint version: 4.1.1

**Comment by JoshuaKGoldberg** [comment]
Should be fixed in **0.1.6**. Thanks for reporting!

**Comment by AmitMY** [comment]
Thanks @JoshuaKGoldberg ! 0.1.6 indeed solves that problem, however there is now a type error, see https://github.com/automutate/autolesshint/issues/65
